pkg/gatherers/conditional/conditional_gatherer_test.go

```diff
@@ -280,15 +281,15 @@ func TestGetGatheringFunctions(t *testing.T) {
 	tests := []struct {
 		name                 string
 		endpoint             string
-		remoteConfig         string
+		remoteConfig         *MockGatheringRulesServiceClient
```

Comment:
nit: I would probably rename also the variable name to something like `mockClient` or `remoteConfigClient`

Reply:
OK, for me, having the `remote` there is helpful because then I easily relate it to the remote server, so I have used `remoteMockClient`. Let me know if you have any other preference.